### PR TITLE
fix(snippet): round new search bar to match other buttons

### PR DESCRIPTION
**Pull request by mohvn**
Updating the search bar design by rounding its corners to match the style of the other buttons in the interface

Before:
before

After:
after

### CODE REVIEW

**Comment by Climax12** [comment]
it's already has rounded corners isn't it?
image

**Comment by mohvn** [comment]
> it's already has rounded corners isn't it? image image
> 
> edit: i was using older version, I think spotify has been changed :) edit2: for me it's still working
> 
> > spotify version: 1.2.47.364.gf06e5cee
> > spicetify version: 2.38.4

image

Not working in my Spotify

Spotify for Windows (64 bit)
1.2.48.405.gf2c48e6f

**Comment by Climax12** [comment]
> > it's already has rounded corners isn't it? image image
> > edit: i was using older version, I think spotify has been changed :) edit2: for me it's still working
> > > spotify version: 1.2.47.364.gf06e5cee
> > > spicetify version: 2.38.4
> 
> image
> 
> Not working in my Spotify
> 
> Spotify for Windows (64 bit) 1.2.48.405.gf2c48e6f
> 
> Spicetify v2.38.4

I just logged in to windows 

ya, Mine also not showing :) 

image

> Spotify for Windows (64 bit) 1.2.48.405.gf2c48e6f
> Spicetify v2.38.4

**Comment by Climax12** [comment]
`.x-filterBox-expandButton, .main-avatar-avatar, .main-topBar-buddyFeed, .main-topBar-button, .Button-sc-1dqy6lx-0, .main-coverSlotCollapsed-expandButton, .NqzueDshzvgXEygqOGPG {border-radius: 15% !important;} .XNFZdOLgMlx491fEWdYJ, .ChipClear__ChipClearComponent-sc-zv5btm-0, .Button-sc-1dqy6lx-0, .Button-sc-y0gtbx-0, .main-nowPlayingView-aboutArtistV2FollowButton, .main-playPauseButton-button, .ButtonInner-sc-14ud5tc-0, .search-searchCategory-carouselButton, .ChipInnerComponent-sm, .ChipInnerComponent-sm-selected, .switch, .arrow-btn, .pSxFsY9Fgcj5f8Gf05mh, .qyKJPLjz8o4jnbk92JOn, .reset, .rdp-button, .btn, .TabItem__StyledTabItem-sc-2ani5y-0  {border-radius: 6px !important;} .view-homeShortcutsGrid-playButton, .ButtonInner-medium-iconOnly,  .button-module__button___hf2qg_marketplace {border-radius: 10px !important;} .ButtonInner-large-iconOnly {border-radius: 12px !important;} .main-globalNav-searchInputContainer, .SFAoASy0S_LZJmYZ3Fh9, .x-searchInput-searchInputInput,  .toggle-module__toggle-indicator-wrapper___6Lcp0_marketplace, .x-toggle-indicatorWrapper, .main-topBar-searchBar {border-collapse: separate; border-radius: 6px !important;}`

add this, it will fix it
since you have opens this pr, so i don't need to add another pr to fix search bar in search page

**Comment by mohvn** [comment]
> `.x-filterBox-expandButton, .main-avatar-avatar, .main-topBar-buddyFeed, .main-topBar-button, .Button-sc-1dqy6lx-0, .main-coverSlotCollapsed-expandButton, .NqzueDshzvgXEygqOGPG, .main-embedWidgetGenerator-themeRadio, .Psc33HXPyazZYAAr1tgz {border-radius: 15% !important;} .XNFZdOLgMlx491fEWdYJ, .ChipClear__ChipClearComponent-sc-zv5btm-0, .Button-sc-1dqy6lx-0, .Button-sc-y0gtbx-0, .main-nowPlayingView-aboutArtistV2FollowButton, .main-playPauseButton-button, .ButtonInner-sc-14ud5tc-0, .search-searchCategory-carouselButton, .ChipInnerComponent-sm, .ChipInnerComponent-sm-selected, .switch, .arrow-btn, .pSxFsY9Fgcj5f8Gf05mh, .qyKJPLjz8o4jnbk92JOn, .reset, .rdp-button, .btn, .TabItem__StyledTabItem-sc-2ani5y-0, .main-playlistEditDetailsModal-closeBtn, .main-embedWidgetGenerator-closeBtn, .Chip__ChipComponent-sc-ry3uox-0 {border-radius: 6px !important;} .view-homeShortcutsGrid-playButton, .ButtonInner-medium-iconOnly, .button-module__button___hf2qg_marketplace {border-radius: 10px !important;} .ButtonInner-large-iconOnly {border-radius: 12px !important;} .main-globalNav-searchInputContainer, .SFAoASy0S_LZJmYZ3Fh9, .x-searchInput-searchInputInput, .toggle-module__toggle-indicator-wrapper___6Lcp0_marketplace, .x-toggle-indicatorWrapper, .main-topBar-searchBar {border-collapse: separate; border-radius: 6px !important;}`
> 
> add this, since you have opened this pr, I not going to make another pr to fix search bar in search page and other minor fixes. 🙂

Thanks, added this fixes to the pr, but still waiting for so long for the review

File: resources/snippets.json [hunk 1]
```diff
@@ -440,7 +440,7 @@
...
     "preview": "resources/assets/snippets/rounded-buttons.png"
   }
-]
+]
```
Comment (theRealPadster) [inline resources/snippets.json:446]: Needs newline at end of file

**Review by theRealPadster — CHANGES_REQUESTED** [review]

File: resources/snippets.json (hunk 1 above)
Comment (mohvn) [reply resources/snippets.json]: got it

**Review by theRealPadster — APPROVED** [review]

**Pull request merged**